tests/declare_type/.testconfig

```diff
@@ -0,0 +1 @@
+cmd: check --all --show-all-errors --strip-root --lib lib
```

Comment:
Actually for tests like this one you don't really need a `.testconfig` file like this (the test runner assumes the absence of one means that you want to test `flow check` on the contents of the dir and compare to the `.exp` file in the dir).

For specifying `--lib lib`, you can just do this in the adjacent `.flowconfig` file:

```
[libs]
lib
```